tests/e2e/util.go

```diff
@@ -2126,3 +2126,60 @@ func getPersistentVolumeClaimSpecForFileShare(namespace string, labels map[strin
 	pvc.Spec.AccessModes = []v1.PersistentVolumeAccessMode{accessMode}
 	return pvc
 }
+
+// waitForPVCToComplete waits for PVC to complete PVC resize. It waits till the condition "FileSystemResizePending" is gone from the PVC condetion status
+func waitForPVCToCompleteResizing(client clientset.Interface, namespace string, pvcName string, poll time.Duration, timeout time.Duration) error {
```

Comment:
why do you need this method, `waitForFSResize()` should do the trick

Reply:
Yes, Modified the code to use waitForFSResize()
Removed waitForPVCToCompleteResizing